app/schemas/recommendation_event.py

```
@@ -0,0 +1,61 @@
+"""
+추천 이벤트 관련 Pydantic 스키마.
+
+recommendation_events 컬렉션에 저장되는 사용자 행동 이벤트 스키마.
+RL 모델의 reward 계산에 사용됩니다.
+"""
+
+from pydantic import BaseModel, Field
+from typing import Optional, List, Dict, Any
+from datetime import datetime
```

Flake8 (7.3.0)

[error] 'datetime.datetime' imported but unused

(F401)